serverless.yml

```diff
@@ -61,6 +61,7 @@ functions:
         - node_modules/**
     events:
       - schedule: cron(0 11 * * ? *) # < every day at 12, rate(1 minute) > every minute
+    timeout: 30
```

Comment:
This _should_ be safe to be minutes long I think? If there are a lot of reminders to send could take more than 30 seconds?

Reply:
sounds reasonable!